src/LSP2ERC725YJSONSchema/encodeVerifiableURI/encodeVerifiableURI.test.ts

```diff
@@ -0,0 +1,26 @@
+import { expect } from 'chai';
+import { concat, hexlify, keccak256, toBeHex, toUtf8Bytes } from 'ethers';
+import { encodeVerifiableURI } from './encodeVerifiableURI';
+
+describe('testing `encodeVerifiableURI`', () => {
+    it('Encoding random json with random url', () => {
+        const json = {
```

Comment:
Maybe you might want to test cases where for instance the input is not a JSON? etc...